app/src/oss/kotlin/com/hippo/ehviewer/cronet/CronetKt.kt

```diff
@@ -1,4 +1,4 @@
-@file:RequiresExtension(extension = Build.VERSION_CODES.S, version = 7)
+@file:Suppress("NewApi")
```

Comment:
The `ChecksSdkIntAtLeast` annotation seems not working for extensions, haven't got time to report to androidx team